install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -582,6 +582,8 @@ spec:
       # In managed etcd mode, Cilium must be able to resolve the DNS name of
       # the etcd service
       dnsPolicy: ClusterFirstWithHostNet
+      {{- else if .Values.dnsPolicy }}
```

Comment:
Should we document that `.Values.dnsPolicy` is ignored if `Values.etcd.managed` is set?

Reply:
updated!